lib/schema.json

```diff
@@ -0,0 +1,167 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "nodes": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "opcode": {
+            "type": "string",
+            "enum": [
```

Comment:
Not sure how the schema could handle:

```
They are going to be consumed and renamed by one of the next stages
```

I think I better change that MUST to SHOULD to make it less confusing.